src/Microsoft.Diagnostics.Monitoring.RestServer/Startup.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Compression;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.ResponseCompression;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Diagnostics.Monitoring.RestServer
```

Comment:
The Startup class also needs to be in the host process (the executable) instead of the RestServer library. Essnetially, this should look like this:

services.AddDiagnosticsServer();
app.UseDiagnosticsServer(); // Since you have a view in here it'll have to use MVC but we should talk about that... 😄